scripts/flag_reviews.py

```diff
@@ -1,66 +1,69 @@
 import pandas as pd
+import numpy as np
 from googleapiclient.discovery import build
-from google_auth_oauthlib.flow import InstalledAppFlow,Flow
+from google_auth_oauthlib.flow import InstalledAppFlow, Flow
 from google.auth.transport.requests import Request
-import os
-import pickle
+from flag import Flag
 from spreadsheets_helper import read_spreadsheet, create_service, export_data_to_sheets
+from database_helper import denormalised_reviews_to_df
 
 def main():
     SCOPES = ['https://www.googleapis.com/auth/spreadsheets']
-    SPREADSHEET_ID_INPUT = '1jFQrYwbhPIaRL6t4TnpTO7W905U0B-W1FXS-GBYwz7M'
     SPREADSHEET_ID_OUTPUT = '1RZDZTswZxI-smwnyi8Oh5ck092hgl_CBxjtQURrczBs'
-    SPREADSHEET_RANGE = 'A1:Z1650'
+    SPREADSHEET_RANGE ='A1:Z3000'
     credentials_file_name = 'credentials.json'
     locality_by_place_id = {}
+    DATABASE_URL = ''
 
-    df = read_spreadsheet(SPREADSHEET_ID_INPUT, SPREADSHEET_RANGE, SCOPES)
-    # df['dt_Timestamp'] = pd.to_datetime(df['Timestamp'])
-    df['flag'] = None
+    df = denormalised_reviews_to_df(DATABASE_URL)
     duplicate_reviews_dict = {}
 
     for index, row in df.iterrows():
-        store_name = row['Store Name']
-        useful_info = row['Useful Information'].strip().lower() if row['Useful Information'] else ''
-        safety_info = row['Safety Observations'].strip().lower() if row['Safety Observations'] else ''
-        timestamp = pd.to_datetime(row['Timestamp'])
+        store_name = row['name']
+        useful_info = row['availabilityInfo'].strip().lower() if row['availabilityInfo'] else ''
+        safety_info = row['safetyInfo'].strip().lower() if row['safetyInfo'] else ''
+        timestamp = pd.to_datetime(row['updatedAt'])
 
-        dict_key = (store_name.strip().lower(), row['User IP'])
+        dict_key = (store_name.strip().lower(), row['ip'])
```

Comment:
i don't think we should use store name here, since i might have updates for 3 different reliance fresh locations from the same IP. place id, or store id are better options.

Reply:
Agreed, will change. I used store name before we had the db.